specification/codesigning/CodeSigning/main.tsp

```diff
@@ -0,0 +1,207 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-autorest";
```

Comment:
```suggestion
```

Your spec should not requires this import.

Reply:
Fixed. Thanks